Directory.Build.props

```diff
@@ -1,14 +1,18 @@
 <Project>
```

Comment:
Note that this file has moved one level up the hierarchy, so that `ManagePackageVersionsCentrally` can be picked up everywhere.